src/content/docs/en/getting-started.mdx

```diff
@@ -102,6 +103,14 @@ Join us in the [Astro Discord](https://astro.build/chat/) to share with and get
 
 [Astro changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
 
+
+<TranslatorList githubRepo="withastro/docs">
+{/* change this HTML heading for a h2 (##) in Markdown */}
```

Comment:
Not sure this is clear that the change should be made _in translations_ — it sounds like the change should happen _in Markdown_, but we’re already in Markdown, so that’s a bit confusing.

Maybe something like this?

```suggestion
{/* Turn this HTML heading into a Markdown h2 — ## — when translating this page. */}
```